ejb/ejb2-connector/ejb2-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/ejb2/checkStateful.java

```diff
@@ -0,0 +1,38 @@
+package org.wso2.carbon.connector.integration.test.ejb2;/*
+* Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Licencing header should go as the first thing in this file. Please check throughout